packages/backend/src/database/migrations/20241021110921_add-timezone-to-scheduler.ts

```diff
@@ -0,0 +1,17 @@
+import { Knex } from 'knex';
+
+const SchedulerTableName = 'scheduler';
+
+export async function up(knex: Knex): Promise<void> {
+    if (await knex.schema.hasTable(SchedulerTableName)) {
+        await knex.schema.alterTable(SchedulerTableName, (table) => {
+            table.string('timezone').nullable();
```

Comment:
We know null/undefined represents UTC timezone. We should define it as the default value in the DB and remove all the `timezone ? timezone : 'UTC'` logic from the codebase.